pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -293,21 +300,31 @@
       }
     },
     "Microsoft.Win32.SystemEvents": {
+      "StableVersions": [
+        "4.5.0"
```

Comment:
Random question, How come some of these don't have a baseline version? Isn't that required for validation when harvesting assets from a package?

Reply:
harvesting is only interested in stable versions. The baseline versions are needed for gathering dependencies and we should update those as well. I will likely update those as well.